OptimizelySDK/Optimizely.cs

```diff
@@ -488,9 +488,15 @@ public virtual bool IsFeatureEnabled(string featureKey, string userId, UserAttri
         /// <param name="variableType">Variable type</param>
         /// <returns>string | null Feature variable value</returns>
         public virtual T GetFeatureVariableValueForType<T>(string featureKey, string variableKey, string userId, 
-            UserAttributes userAttributes, FeatureVariable.VariableType variableType, ProjectConfig config)
+            UserAttributes userAttributes, FeatureVariable.VariableType variableType)
         {
 
+            var config = ProjectConfigManager?.GetConfig();
+            if (!IsValid && config == null) {
+                Logger.Log(LogLevel.ERROR, "Datafile has invalid format. Failing 'activate'.");
```

Comment:
This still refers to `activate`. Please make sure it references the right API name.